docs/source/app/holography.rst

```diff
@@ -0,0 +1,195 @@
+Off-axis electron holography
+============================
+
+Hologram simulation
+-------------------
+Holograms can be simulated using the method described by Lichte et al.
```

Comment:
You can add a reference to `docs/source/references-libertem.bib` and use :cite:`<key>` here to cite the reference.